Dockerfile

```diff
@@ -38,6 +36,8 @@ RUN cd node_modules/classic-level && yarn rebuild
 FROM node:20-alpine
 WORKDIR /usr/app
 COPY --from=build_deps /usr/app .
+# Explicitly remove yarn offline mirror. Can't be added in .dockerignore as it's used in previous build stages
+RUN rm -fr /usr/app/.yarn/
```

Comment:
this will add another layer to the final image, can't you delete this at the end of the previous stage?

Reply:
Done